steward/src/gas.rs

```diff
@@ -52,4 +59,19 @@ impl CellarGas {
         let data = etherchain_oracle.fetch().await;
         data
     }
+
+    pub fn pad(price: U256) -> Result<U256, Error> {
```

Comment:
Nit: I don't think the term "pad" is accurate for multiplying by a floating point.